Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -261,3 +207,33 @@
     @ChildContent
 </CascadingValue>
 
+@code {
+
+    private RenderFragment _Pagination => __builder =>

```

Comment:
Prefer member vars without underscore. Also, rename them to PaginationFragment, or PaginationRenderFragment, or something similar...